CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.6)
+
+if (DEFINED ENV{VCPKG_ROOT})
+  set(CMAKE_TOOLCHAIN_FILE "$ENV{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake")
+endif ()
+
+set(n clinfo)
+project(${n})
+
+find_package(OpenCL REQUIRED)
+
+add_executable(${n} src/${n}.c)
+target_include_directories(${n} PUBLIC src)
+target_include_directories(${n} PUBLIC ${OpenCL_INCLUDE_DIRS})
+target_link_libraries(${n} ${OpenCL_LIBRARIES} ${CMAKE_DL_LIBS})
```

Comment:
If we require CMake 3.7 we could use an imported target instead and get rid of that include directory:

```cmake
target_link_libraries(${n} PRIVATE OpenCL::OpenCL)
```

In any case, always use the PRIVATE/PUBLIC/INTERFACE keywords with `target_link_libraries`. It is [incompatible](https://cmake.org/cmake/help/latest/command/target_link_libraries.html#id4) with the new syntax.